addon/components/vertical-collection/component.js

```diff
@@ -57,33 +93,48 @@ const VerticalCollection = Component.extend({
   containerSelector: null,
 
   // –––––––––––––– Performance Tuning
-  /*
-   * how much extra room to keep visible and invisible on
-   * either side of the viewport.
+  /**
+   * The amount of extra room in pixels to keep visible and invisible on
+   * either side of the viewport. This value needs to be greater than 0.
```

Comment:
this used to be `pixels` but it is now a strict Item count I believe cc @pzuraq

Reply:
Yep, strict item count